### PR TITLE
docs: add docs edit link in meta

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/48004#issuecomment-2014565788

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      --     |
| 🇨🇳 Chinese |     --      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/builtins/ComponentMeta/index.tsx [hunk 1]
```diff
@@ -143,7 +157,19 @@ const ComponentMeta: React.FC<ComponentMetaProps> = (props) => {
...
+          },
+          filename && {
+            label: locale.docs,
+            children: (
+              <Typography.Link
+                className={styles.code}
+                href={`${branchUrl}${filename}`}
+                target="_blank"
+              >
+                <EditOutlined style={{ marginInlineEnd: 3 }} /> {locale.edit}
```
Comment (li-jia-nan) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:172]: 这个最好用 4 的倍数，按照设计规范

Comment (afc163) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:172]:
```suggestion
                <EditOutlined style={{ marginInlineEnd: 4 }} />{locale.edit}
```

**Review by li-jia-nan — DISMISSED** [review]

File: .dumi/theme/builtins/ComponentMeta/index.tsx [hunk 2]
```diff
@@ -143,7 +157,19 @@ const ComponentMeta: React.FC<ComponentMetaProps> = (props) => {
...
+          },
+          filename && {
+            label: locale.docs,
+            children: (
+              <Typography.Link
+                className={styles.code}
+                href={`${branchUrl}${filename}`}
+                target="_blank"
+              >
+                <EditOutlined style={{ marginInlineEnd: 4 }} />{locale.edit}
```
Comment (afc163) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:172]:
```suggestion
                <EditOutlined style={{ marginInlineEnd: 4 }} />
                <span>{locale.edit}</span>
```

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
预览链接又坏了：

image

**Comment by li-jia-nan** [comment]
上面这个可以去掉了吧：

image

**Comment by afc163** [comment]
不能去，普通文档页没有这个 meta。

**Pull request merged**